### PR TITLE
Update HttpClientWrapper to WebServiceDriver

### LINKED ISSUE #116: Rename HttpClientWrapper to WebServiceDriver
### Review
- [x]  Rename HttpClientWrapper to WebServiceDriver
- [x] Add WebServiceDriver constructor to take in a ClosableHttpClient to set the baseHttpClient
- [x] Change wrapper to Driver where needed
- [x] Check all tests pass after name change

### Branches/PRs
#117
### Notes
This issue is strictly to change the Client Wrapper to a driver and any of the changes that go with it. This will impact several classes and test pages.

### PR DESCRIPTION
**Pull request by jonreding2010**
Renamed HttpClientWrapper to WebServiceDriver.
Renamed WebServiceWithWrapperTest to WebServiceDriverUnitTest.
Updated classes to use WebServiceDriver.
Updated Tests to use WebServiceDriver.

This closes #116.

### CODE REVIEW

File: Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/WebServiceTestObject.java [hunk 1]
```diff
@@ -13,42 +13,36 @@
...
-   * @param logger
-   *          The Logger Object
-   * @param fullyQualifiedTestName
-   *          The fully qualified test name
+   * @param newWebServiceDriver The web service driver.
+   * @param logger The Logger Object.
+   * @param fullyQualifiedTestName The fully qualified test name.
    */
-  public WebServiceTestObject(HttpClientWrapper webServiceWrapper, Logger logger, String fullyQualifiedTestName) {
+  public WebServiceTestObject(final WebServiceDriver newWebServiceDriver,
```
Comment (bradyrad) [inline Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/WebServiceTestObject.java]: I don't think it's necessary to declare param as final. @jason-edstrom thoughts?

File: Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/WebServiceTestObject.java [hunk 2]
```diff
@@ -13,42 +13,36 @@
...
   /**
    * Set the web service wrapper for the WebServiceTestObject.
-   * 
-   * @param webServiceWrapper
-   *          The web service wrapper object
+   * @param newWebServiceDriver The web service driver
    */
-  public void setWebServiceWrapper(HttpClientWrapper webServiceWrapper) {
-    this.webServiceWrapper = webServiceWrapper;
+  public void setWebServiceDriver(final WebServiceDriver newWebServiceDriver) {
```
Comment (bradyrad) [inline Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/WebServiceTestObject.java]: I don't think it's necessary to declare param as final.

File: Framework/jmaqs-webservices/src/test/java/com/magenic/jmaqs/webservices/WebServiceDriverUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+      @Override
+      public void close() { }
+
+      @Override
+      protected CloseableHttpResponse doExecute(HttpHost httpHost,
+              HttpRequest httpRequest, HttpContext httpContext) {
+        return null;
+      }
+    };
```
Comment (bradyrad) [inline Framework/jmaqs-webservices/src/test/java/com/magenic/jmaqs/webservices/WebServiceDriverUnitTest.java]: See the getHttpClient method in WebServiceDriver and use that to create the CloseableHttpClient. I would suggest using different Media Types in the Header object when creating and setting the CloseableHttpClient.

File: Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java [hunk 4]
```diff
@@ -22,12 +21,12 @@
   private ThreadLocal<WebServiceTestObject> webServiceTestObject = new ThreadLocal<WebServiceTestObject>();
```
Comment (jason-edstrom) [inline Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java]: Granted, this is just for the rename, but we could hit this with a deprecation tag now or later.  Either way works.

Comment (jason-edstrom) [reply Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java]: Probably worth just getting it in there now.

File: Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java [hunk 5]
```diff
@@ -47,7 +46,7 @@ public WebServiceTestObject getWebServiceTestObject() {
   @Override
   protected void postSetupLogging() throws Exception {
```
Comment (jason-edstrom) [inline Framework/jmaqs-webservices/src/main/java/com/magenic/jmaqs/webservices/BaseWebServiceTest.java]: Deprecate this as well

**Pull request merged**